x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/execution_command_shell_started_by_svchost.json

```diff
@@ -45,5 +45,5 @@
   "timeline_title": "Comprehensive Process Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 9
+  "version": 10
```

Comment:
This version was already incremented from `8` to `9` in `8.2`, see: https://github.com/elastic/kibana/pull/129526/files#diff-8b7ec1d27abcbdb4894cf73ea6a91fef47bd29c0fbd8886ed7e9a658035e4200R48